### PR TITLE
Changes done with UI and Bug solved. 

### PR DESCRIPTION
**Pull request by PedroTrujilloV**
Bug Snapshot “What is happening is you have managed to go back to the first view controller right as the location service was updating the user's location and taking a new snapshot. As the snapshot finished rendering the viewcontroller was being deallocated and so the return block was losing a reference to self. This is an excellent bug discovery! Thank you for finding it!”: Tom
- Solved by Tom.
- Tested UI for alert controller.

### CODE REVIEW

File: Street Trees.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1430,6 +1436,7 @@
 				PRODUCT_BUNDLE_IDENTIFIER = "com.CodeForOrlando.Street-Trees";
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				PROVISIONING_PROFILE = "";
+				SWIFT_VERSION = 2.3;
```
Comment (LondonAtlas) [inline Street Trees.xcodeproj/project.pbxproj:1445]: this needs to be changed back to 2.2

File: Street Trees.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -1446,6 +1453,7 @@
...
```
Comment (LondonAtlas) [inline Street Trees.xcodeproj/project.pbxproj:1462]: this needs to be changed back to 2.2

File: Street Trees/STConfirmationPageViewController.swift [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+    {
+        
+        if self.confirm()
+        {
+            
+
+            let alertConfimationMessage = UIAlertController(title: "Confirmation\n\n\n\n", message: "Thank You! Your request has been sent.", preferredStyle: .Alert)
+            
+            
+            let sentImage = UIImage(named: "sent")!.imageWithRenderingMode(UIImageRenderingMode.AlwaysOriginal)
```
Comment (LondonAtlas) [inline Street Trees/STConfirmationPageViewController.swift]: Never force unwrap anything other than IBOutlets.

File: Street Trees/STConfirmationPageViewController.swift [hunk 4]
```diff
@@ -0,0 +1,251 @@
...
+            let imageWidth = sentImage.size.width * 0.6
+            let imageHeight = sentImage.size.height * 0.6
+            let marginLeft:CGFloat = (alertConfimationMessage.view.bounds.size.width * 0.5) - (imageWidth * 0.72)
+            let marginTop:CGFloat = 42.0
+            let rect = CGRectMake(marginLeft, marginTop, imageWidth, imageHeight)
+            let imageView = UIImageView(frame: rect)
+            imageView.image = sentImage
+            alertConfimationMessage.view.addSubview(imageView)
```
Comment (LondonAtlas) [inline Street Trees/STConfirmationPageViewController.swift]: The image and all of this custom framing doesn't look good on the alert. If you want to use the confirmation image, I would create a new view controller and push that on, or remove the image all together.

Second reason to not use an image in an alert view is that you will need to create custom logic for every screen size as it varies from 4S to iPad Pro.

File: Street Trees/STConfirmationPageViewController.swift [hunk 5]
```diff
@@ -0,0 +1,251 @@
...
+            let okAction = UIAlertAction(title: "Ok", style: .Default, handler: nil)
```
Comment (LondonAtlas) [inline Street Trees/STConfirmationPageViewController.swift]: The OK action to dismiss the order form back to the map view.

**Review by LondonAtlas — CHANGES_REQUESTED** [review]
This is coming together very well! :D

File: Street Trees.xcodeproj/project.pbxproj (hunk 2 above)
Comment (PedroTrujilloV) [reply Street Trees.xcodeproj/project.pbxproj:1462]: Done

File: Street Trees/STConfirmationPageViewController.swift (hunk 3 above)
Comment (PedroTrujilloV) [reply Street Trees/STConfirmationPageViewController.swift]: Done

File: Street Trees/STConfirmationPageViewController.swift (hunk 5 above)
Comment (PedroTrujilloV) [reply Street Trees/STConfirmationPageViewController.swift]: Done

File: Street Trees/STConfirmationPageViewController.swift (hunk 4 above)
Comment (PedroTrujilloV) [reply Street Trees/STConfirmationPageViewController.swift]: Done.

**Pull request closed without merging**